src/MeetingsSDKAdapter.js

```diff
@@ -181,7 +183,15 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
         this.meetings[ID] = {...meeting, localShare: stream};
         break;
       case MEDIA_TYPE_REMOTE_SHARE:
-        this.meetings[ID] = {...meeting, remoteShare: stream};
+        this.meetings[ID] = {...meeting, remoteShareStream: stream};
+        break;
+      case EVENT_REMOTE_SHARE_START:
+        // Only activate the remote stream when get get the start notification
+        this.meetings[ID] = {...meeting, remoteShare: meeting.remoteShareStream};
```

Comment:
We also have the option to create a new stream every time from the same track. I figured the way it was implemented cause less object creation and memory consumption:

```js
this.meetings[ID] = {...meeting, remoteShare: new MediaStream([meeting.mediaProperties.remoteShare])};
```